packages/list-plugins/lib/tracking/atTracking.js

```diff
@@ -0,0 +1,69 @@
+const { DateTime } = require('@keystone-alpha/fields');
+
+const composeResolveInput = (originalHook, newHook) => async params => {
+  const resolvedData = await originalHook(params);
+  return newHook({ ...params, resolvedData });
+};
+
+const _atTracking = ({ created = true, updated = true }) => ({
+  updatedAtField = 'updatedAt',
+  createdAtField = 'createdAt',
+  ...atFieldOptions
+} = {}) => ({ fields = {}, hooks = {}, ...rest }) => {
+  const dateTimeOptions = {
+    type: DateTime,
+    format: 'MM/DD/YYYY h:mm A',
+    access: {
+      read: true,
+      create: true, // TODO: revert to false when read only fields are available
```

Comment:
someone can inadvertently choose the field and try update that in admin-ui as this is available even if we say `false` here. This breaks admin ui creation of item with error 400. admin-ui sends graphql mutation with the field and that is not added to schema.

Reply:
Rather than shipping this with todo comments let's fix the issue that is holding us back.
I need to understand it better. Is there a current issue open with a simple test case?

Reply:
there is no specific issue for this, but there is on open for simple feature https://github.com/keystonejs/keystone-5/pull/1286

Reply:
This is now resolved